# Autol LiDAR packet.
uint8[954] data	# packet data
